syntax = "proto3";

package devices;

service ManageDevice {
    rpc command (CommandRequest) returns (CommandReply) {}
}

message CommandRequest {
    string device_name = 1;
    string order = 2;
    int32 value = 3;
}

message CommandReply {
    string device_name = 1;
    string response = 2;  
}
